ValidationRelay/Relay.swift

```diff
@@ -20,15 +20,28 @@ func getIdentifiers() -> [String: String] {
         }
     }
     
+    guard let build = buildNumber() else {
+        print("Failed to retrieve build number")
+        return [:]
+    }
+    
+    guard let uniqueDeviceID = MGCopyAnswer("UniqueDeviceID" as CFString)?.takeRetainedValue() as? String else {
```

Comment:
This stuff should never fail: if it did, `absd` would have been unable to generate validation data (it uses the same API